scrabble-scorer.js

```diff
@@ -33,26 +33,71 @@ function oldScrabbleScorer(word) {
 // don't change the names or your program won't work as expected. //
 
 function initialPrompt() {
-   console.log("Let's play some scrabble! Enter a word:");
-};
+   let userInput = input.question("Let's play some scrabble!\n\nEnter a word: ");
+   return userInput;
+}
 
-let simpleScore;
+let simpleScore = {
+  name: 'Simple Score',
+  description: 'Each letter is worth 1 point.',
+  scoreFunction: function(wordToGrade) {return wordToGrade.length;}
+};
```

Comment:
should just be an anonymous  function
`let simpleScore=function(wordToGrade){}`